src/styles/block/calendar.scss

```
@@ -0,0 +1,58 @@
+.calendar {
+
+  display: flex;
+  flex-wrap: wrap;
+
+  width: 7 * $day__size + $month__paddings + $gap__days * 6;
+  padding: $month__paddings;
+  gap: $gap__days;
+  
+  &__day {
+    box-sizing: border-box;
+    height: $day__size;
+    width: $day__size;
+    background-color: #eee;
+    border: $border__days solid black;
+    cursor: pointer;
+    transition: 0.2s linear 0.1s;
```

Always shoy which rules you apply transition to.
And also, task said that transition duration should be 0.5s.
```suggestion
    transition: transform 0.5s, background-color 0.5s;
```